azure-devops/azext_devops/dev/boards/arguments.py

```diff
@@ -20,3 +20,14 @@ def load_work_arguments(self, _):
     with self.argument_context('boards work-item delete') as context:
         context.argument('yes', options_list=['--yes', '-y'], action='store_true',
                          help='Do not prompt for confirmation.')
+
+    with self.argument_context('boards work-item relation') as context:
+        context.argument('id', help='The ID of the work item')
+        context.argument('target_ids', help='ID(s) of work-items to create relation with. \
```

Comment:
Can you ensure the parameter is in the singular - target-id?

Reply:
But it can accept multiple values. We have this at other places as well. 
![image](https://user-images.githubusercontent.com/11888714/56366202-303fd800-6210-11e9-87e7-c244f5718739.png)

Reply:
😟 - Singular is a parameter paradigm - can't locate the documentation location but I remember from a conversation with the Azure CLI team; we'll need to correct this as well.